tests/TestCase/Error/DebuggerTest.php

```diff
@@ -272,7 +272,7 @@ public function testExportVar()
 	theme => null
 	hasRendered => false
 	uuids => []
-	request => object(Cake\Network\Request) {}
+	request => object(Cake\Http\ServerRequest) {}
 	response => object(Cake\Network\Response) {}
```

Comment:
The Psr interface name is ResponseInterface and Diactoros class name is also Response. So neither uses "Server" prefix.